utils/useGetZoomLink.ts

```diff
@@ -0,0 +1,20 @@
+import useSWR from "swr";
+import fetcher from "./fetcher";
+
+export type UseGetZoomLinkResponse = {
+    link: string;
+    isLoading: boolean;
+    error: any;
+};
+
+/**
+ * Me hook to get data about current user
+ */
+export default function useGetZoomLink(): UseGetZoomLinkResponse {
+    const { data, error } = useSWR("/api/class/join-class", fetcher);
```

Comment:
idem, join-class always sounds like a PUT method right? If we rename the api file to `meeting.ts`, it would look like:
```suggestion
    const { data, error } = useSWR("/api/class/meeting", fetcher);
```